neutron_vpnaas/services/vpn/device_drivers/ovn_ipsec.py

```diff
@@ -183,12 +184,18 @@ def delete(self, name):
             LOG.exception(msg, name)
 
 
-class OvnOpenSwanProcess(ipsec.OpenSwanProcess):
-    pass
+class OvnOpenSwanProcess(ipsec.OpenSwanProcess, meter_ipsec.MeterProcess):
+    
+    def __init__(self, conf, process_id, vpnservice, namespace):
+        super(OvnOpenSwanProcess).__init__(conf, process_id, vpnservice, namespace)
+	meter_ipsec.MeterProcess.__init__(self, conf, process_id, vpnservice, namespaces)
```

Comment:
if we inherit from MeterProcess, super function should initialize MeterProcess too, right?
So this line is not needed?